src/main/_data/sidebars/che_7_docs.yml

```diff
@@ -234,6 +234,9 @@ entries:
   - title: Che extensibility reference
     url: che-7/che-extensibility-reference
     output: web
+  - title: Language Server Protocol reference
```

Comment:
@themr0c, I think this would be better as a sub-item under the "Che ext. reference" topic. If you agree, please:
* create an 'Overview' page for "Che ext. ref." with links to all sub-items; see, e.g. https://www.eclipse.org/che/docs/che-7/retrieving-che-logs/
* modify this sidebar yaml to have an expandable "Che ext. ref." item with `folderitems`

Reply:
Great idea. 
Given the fact that such a structural change would apply to Debug Adapter Protocol reference as well (https://github.com/eclipse/che-docs/pull/815), it will be easier to handle with one single PR consolidating the changes for Debug Adapter Protocol reference and Language Server Protocol reference.